HW14/tests/HW14.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation     A test cases for HW14
+Library     SeleniumLibrary
+Library     ../libraries/pageobjects/MonitorsPage.py
+Library     ../libraries/pageobjects/CartPage.py
+Variables   ../libraries/variables.py
+
+*** Variables ***
+${URL}  https://www.demoblaze.com/
+${USERNAME}     ArtemB
+${PASSWORD}     ArtemB
+
+*** Test Cases ***
+hw_part_1
+    Open Browser    ${URL}   chrome
+    Login with valid credentials    ${USERNAME}     ${PASSWORD}
+    Element Should Be Visible   ${LOGOUT_BUTTON}
+    Element Text Should Be  ${WELCOME_MESSAGE_TEST}  Welcome ${USERNAME}
+
+hw_part_2
+    Open Browser    ${URL}   chrome
+    Login with valid credentials    ${USERNAME}     ${PASSWORD}
+    Click Element   ${MONITORS_BUTTON}
+    BuiltIn.Sleep   2
```

Comment:
it's better to avoid using direct 'sleep's
Wait Until Keyword Succeeds built-in rf keyword might be useful

Reply:
makes sense. 
Reworked